Resources/views/standard_layout.html.twig

```diff
@@ -142,7 +142,13 @@ file that was distributed with this source code.
                                                         {% if not loop.last  %}
                                                             <li>
                                                                 {% if menu.uri is not empty %}
-                                                                    <a href="{{ menu.uri }}">{{ menu.label|raw }}</a>
+                                                                    <a href="{{ menu.uri }}">
+                                                                        {% if menu.extra('safe_label', false) %}
```

Comment:
Should not be the reverse?

Or use `safe_html` option like twig does to avoid confusion?